commerce-frontend-js/src/main/resources/META-INF/resources/components/assigner/Assigner.es.js

```diff
@@ -8,8 +8,8 @@ function Assigner(props) {
 
 	return (
 		<ClayIconSpriteContext.Provider value={props.spritemap}>
-			<div className="d-xl-flex align-items-center px-3 mr-3 border-right">
-				<span className="mr-3">Assigned to:</span>
+			<div className="d-flex align-items-center">
+				<span className="mr-3">{Liferay.Language.get("assigned-to")}:</span>
```

Comment:
We shouldn't append even a colon to this localization -- if it should be appended, it should be part of the localization.